src/DatePicker/Utilities.elm

```diff
@@ -314,3 +317,69 @@ toUtcDateTimeString datetime =
         ++ ":"
         ++ addLeadingZero (Time.toSecond Time.utc datetime)
         ++ " (UTC)"
+
+
+setDayNotTime : Maybe Posix -> Posix -> Posix
```

Comment:
You should also add documentation. I don't understand what happens if you pass in `Nothing`. I feel like allowing `Nothing` is not a good idea here but maybe if you explain what the defined behavior is that case it makes sense.

Reply:
It looks like you are passing in a default anyway so maybe you can just remove the Maybe.